docs/installation_configuration_guide/Installation.md

```diff
@@ -197,3 +182,46 @@ Available Nussknacker image environment variables below. $NUSSKNACKER_DIR is a p
 | $NUSSKNACKER_DIR/components              | Can be used in MODEL_CLASS_PATH property                | Directory with Nussknacker Component Provider JARS                                                                                            |
 | $NUSSKNACKER_DIR/lib                     |                                                         | Directory with Nussknacker base libraries                                                                                                     |
 | $NUSSKNACKER_DIR/managers                |                                                         | Directory with Nussknacker Deployment Managers                                                                                                |
+
+# Snippets
```

Comment:
Sample configurations? 
And maybe some comments about what they do? Especially with the nginx?

Reply:
added some description